OpenRA.Game/Settings.cs

```diff
@@ -171,7 +171,7 @@ public class GameSettings
 
 		public bool UseClassicMouseStyle = false;
 		public bool AlwaysShowStatusBars = false;
-		public bool TeamHealthColors = false;
+		public bool AllianceDisplay = false;
```

Comment:
How about `UseAllianceColors` with `ToggleAllianceColorsKey` for the hotkey?  This makes it a bit more self descriptive.